CONTRIBUTING.md

```diff
@@ -1,61 +1,48 @@
 # Contributing to alertmanager-k8s
-Alertmanager, as the name suggests, filters incoming alerts and routes them to
-pre-defined receivers. Alertmanager reads its configuration from an
-`alertmanager.yml`, some aspects of which are exposed to the user via
-`juju config` calls (see [`config.yaml`](config.yaml)).
-In the future, integrator charms may be used for configuring Alertmanager.
-
-The intended use case of this operator is to be deployed together with the
-[prometheus-k8s operator][Prometheus operator], although that is not
-necessary, as [Alertmanager's HTTP API][Alertmanager API browser] could be
-[used](https://github.com/prometheus/alertmanager/issues/437#issuecomment-263413632)
-instead.
-
-## Known issues
-1. Adding multiple receivers of the same type (e.g. PagerDuty) is not very scalable, due to the
-   nature of the `juju config` command. This is likely to improve in the future by using integrator
-   charms.
-
-## Bugs and pull requests
-- Generally, before developing enhancements to this charm, you should consider
-  explaining your use case.
-- If you would like to chat with us about your use-cases or proposed
-  implementation, you can reach us at
-  [Canonical Mattermost public channel](https://chat.charmhub.io/charmhub/channels/charm-dev)
-  or [Discourse](https://discourse.charmhub.io/).
-- All enhancements require review before being merged. Apart from
-  code quality and test coverage, the review will also take into
-  account the resulting user experience for Juju administrators using
-  this charm.
+
+## Overview
+
+This documents explains the processes and practices recommended for
+contributing enhancements or bug fixing to the Alertmanager Charmed Operator.
+
+The intended use case of this operator is to be deployed as part of the
+[COS Lite] bundle, although that is not necessary.
+
 
 ## Setup
 
 A typical setup using [snaps](https://snapcraft.io/) can be found in the
 [Juju docs](https://juju.is/docs/sdk/dev-setup).
 
+
 ## Developing
 
-Use your existing Python 3 development environment or create and
-activate a Python 3 virtualenv
+- Prior to getting started on a pull request, we first encourage you to open an
+  issue explaining the use case or bug.
+  This gives other contributors a chance to weigh in early in the process.
+- To author PRs you should be familiar with [juju](https://juju.is/#what-is-juju)
+  and [how operators are written](https://juju.is/docs/sdk).
+- The best way to get a head start is to join the conversation on our
+  [Mattermost channel] or [Discourse].
```

Comment:
Link to Mattermost and Discourse?

Reply:
It's linked! This is a short-form reference syntax. The link are at the bottom.